docs/insights/architecture.md

```diff
@@ -190,6 +190,113 @@ This sniff reports use of superfluous prefix or suffix "Trait" for traits.
 
 **Insight Class**: `SlevomatCodingStandard\Sniffs\Classes\SuperfluousTraitNamingSniff`
 
+## Ordered class elements <Badge text="^1.8"/> <Badge text="Architecture\Classes" type="warn"/> <Badge text="configurable"/>
+
+This sniff orders the elements of classes/interfaces/traits.
+
+**Insight Class**: `PhpCsFixer\Fixer\ClassNotation\OrderedClassElementsFixer`
+
+<details>
+    <summary>Configuration</summary>
+
+```php
+\PhpCsFixer\Fixer\ClassNotation\OrderedClassElementsFixer::class => [
+    'order' => [ // List of strings defining order of elements.
+        'use_trait',
+        'constant_public',
+        'constant_protected',
+        'constant_private',
+        'property_public',
+        'property_protected',
+        'property_private',
+        'construct',
+        'destruct',
+        'magic',
+        'phpunit',
+        'method_public',
+        'method_protected',
+        'method_private',  
+    ],
+    'sortAlgorithm' => 'none' // possible values ['none', 'alpha']
+]
+```
+</details>
+
+## Single class element per statement <Badge text="^1.8"/> <Badge text="Architecture\Classes" type="warn"/> <Badge text="configurable"/>
+
+There must not be more than one property or constant declared per statement.
+
+**Insight Class**: `PhpCsFixer\Fixer\ClassNotation\SingleClassElementPerStatementFixer`
+
+<details>
+    <summary>Configuration</summary>
+
+```php
+\PhpCsFixer\Fixer\ClassNotation\SingleClassElementPerStatementFixer::class => [
+    'elements' => [
+        'const',
+        'property',
+    ],
+]
+```
+</details>
+
+## Ordered imports <Badge text="^1.8"/> <Badge text="Architecture\Classes" type="warn"/> <Badge text="configurable"/>
+
+This sniff order `use` statements.
+
+**Insight Class**: `PhpCsFixer\Fixer\Import\OrderedImportsFixer`
+
+<details>
+    <summary>Configuration</summary>
+
+```php
+\PhpCsFixer\Fixer\Import\OrderedImportsFixer::class => [
+    'import_order' => null, // or [ 'class', 'const', 'function']
```

Comment:
```suggestion
    'import_order' => null, // possible values [ null, 'class', 'const', 'function']
```

Reply:
In fact, values possible are `null` or an array with custom order between `class`, `const`, and `function`

The `null` parameter will apply default order = `['class', 'const', 'function']` 

See https://mlocati.github.io/php-cs-fixer-configurator/#version:2.15|fixer:ordered_imports 

Maybe we should add default order directly in array, WDYT @olivernybroe ?

Reply:
Oh, thanks, makes sense. 

I think adding a default order makes sense, this way we won't get weird results if they change their default order as we have our own.